flash/vision/backbones.py

```diff
@@ -11,11 +11,71 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from contextlib import suppress
 from typing import Tuple
 
-import torch.nn as nn
 import torchvision
+from pytorch_lightning.utilities import _BOLTS_AVAILABLE
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from torch import nn as nn
+
+if _BOLTS_AVAILABLE:
+    with suppress(TypeError):
```

Comment:
What is this for? It shouldn'f fail if _BOLTS_AVAILABLE

Reply:
LGTM otherwise

Reply:
You could remove the `suppress` part.

Reply:
Yeah I'm not sure what this was for, it was in the code used to load the bolts models for image embedding